CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This file is influenced by http://keepachangelog.com/.
 ### Changed
 - Incorrect attempts at verifying identity locks the user out
 - Change git merge strategy for CHANGELOG.md to reduce conflicts
+- Update News article page template
```

Comment:
It might be worth including what was changed - i.e. "including publishers and related news articles and topics"